src/quo2/components/tags/context_tag/view.cljs

```diff
@@ -40,31 +40,39 @@
     (trim-public-key public-key)]])
 
 (defn context-tag
-  [{:keys [text-style blur? no-avatar-placeholder?] :as params} photo name channel-name]
-  (let [text-params       {:weight :medium
-                           :size   :paragraph-2
-                           :style  (assoc text-style :justify-content :center)}
+  [{:keys [text-style blur? no-avatar-placeholder? text-container-style ellipsize-text?]
+    :as   props}
```

Comment:
To fix this problem I preferred to make this component more flexible, so now receives more props.

This should be addressed in #16577, when we have all variants properly implemented